openfisca_core/taxbenefitsystems.py

```diff
@@ -74,6 +76,15 @@ def base_tax_benefit_system(self):
             self._base_tax_benefit_system = base_tax_benefit_system = baseline.base_tax_benefit_system
         return base_tax_benefit_system
 
+    def instantiate_entities(self):
+        person_instance = self.person_entity(None)
+        entities_instances: Dict[Entity.key, Entity] = {person_instance.key: person_instance}
```

Comment:
> What does `Entity.key` mean in a type annotation? I would have expected `Dict[String, Entity]`, but I'm not familiar with them. Any reference appreciated 🙂

Reply:
@fpagnoux I feel a bit naughty about this, but it's for a good cause. We considered `Dict[str, Entity]` which is technically correct, but since these annotations are not checked, we can be more "poetic": so we use `Dict[Entity.key, Entity]` to distinguish from `Dict[Entity.plural, Entity]` because that's one of the most likely confusions. It's possible that the syntax is itself a source of confusion and surprise, and thus the decision is a net negative.

Reply:
> 👌